### PR TITLE
[KO] Additional Translation of Basics-Basics lesson.

### PR DESCRIPTION
**Pull request by devleoper**
Not only materials added #480, There are also updated stuff since I translated basic lesson.

Don't hesitate to review changes things. There were some word changes in English, I changed some wordings in Korean.

### CODE REVIEW

**Comment by devleoper** [comment]
And it can be also kind of Korean Translation(#201).
Therefore I need your help @Dalgona, @malkoG, @riseshia :wink:

File: ko/lessons/basics/basics.md [hunk 1]
```diff
@@ -6,28 +6,46 @@ order: 1
...
 
-	Erlang/OTP 17 [erts-6.4] [source] [64-bit] [smp:8:8] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
+    Erlang/OTP {{ site.erlang.OTP }} [erts-{{ site.erlang.erts }}] [source] [64-bit] [smp:4:4] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
+
+    Interactive Elixir ({{ site.elixir.version }}) - press Ctrl+C to exit (type h() ENTER for help)
+    iex>
 
-	Interactive Elixir ({{ site.elixir.version }}) - press Ctrl+C to exit (type h() ENTER for help)
-	iex>
+여기서 계속 나아가봅시다. 간단한 코드를 조금 써 보면서 한번 체험해보세요.
```
Comment (riseshia) [inline ko/lessons/basics/basics.md]: (maybe) `한번` seems to be unnecessary

File: ko/lessons/basics/basics.md [hunk 2]
```diff
@@ -6,28 +6,46 @@ order: 1
...
 
-## 기본 타입
+    iex(1)> 2+3
+    5
+    iex(2)> 2+3 == 5
+    true
+    iex(3)> String.length("The quick brown fox jumps over the lazy dog")
+    43
+
+여기서 입력해 본 모든 코드를 이해하지 못하더라도 벌써부터 걱정하지는 마시고, 어떤 느낌인지 감만 잡아보세요.
```
Comment (riseshia) [inline ko/lessons/basics/basics.md]: (maybe) `걱정하지는 마시고` -> `걱정하지 마시고`

File: ko/lessons/basics/basics.md [hunk 3]
```diff
@@ -6,28 +6,46 @@ order: 1
...
 
 {% include toc.html %}
 
-## 준비
+## 시작하기
 
 ### Elixir 설치하기
 
-Elixir-lang.org 홈페이지의 [Installing Elixir](http://elixir-lang.org/install.html) 가이드에서 각 운영체제별로 설치하는 방법을 알아볼 수 있습니다.
+elixir-lang.org 홈페이지의 [Installing Elixir](http://elixir-lang.org/install.html) 가이드에서 각 운영체제별로 설치하는 방법을 알아볼 수 있습니다.
```
Comment (riseshia) [inline ko/lessons/basics/basics.md]: (maybe) `각 운영체제별로` -> `운영체제별로`

**Comment by malkoG** [comment]
Nice 😸

File: ko/lessons/basics/basics.md [hunk 4]
```diff
@@ -117,6 +149,8 @@ iex> "foo\nbar"
 "foo\nbar"
 ```
 
+이보다 더 복잡한 데이터 타입이 Elixir에 더 있습니다. 이런 부분은 컬렉션이나 함수를 다룰 때 조금 더 알아보도록 하겠습니다.
```
Comment (riseshia) [inline ko/lessons/basics/basics.md]: (maybe) `이보다 더 복잡한 데이터 타입이 Elixir에 더 있습니다.` -> `Elixir에는 이보다 더 복잡한 데이터 타입도 있습니다.`

**Comment by riseshia** [comment]
I'm done XD

File: ko/lessons/basics/basics.md (hunk 1 above)
Comment (devleoper) [inline ko/lessons/basics/basics.md]: Hmm I agree. removing redundant expressions sounds pretty good. :smiley:

**Pull request merged**